src/transformStateWithClones.js

```diff
@@ -65,7 +65,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  let stateCopy = { ...state };
+  const transformChanges = [];
+
+  for (const transform of transforms) {
+    stateCopy = { ...stateCopy };
+    transformChanges.push(stateCopy);
+
+    switch (transform.operation) {
```

Comment:
usually in switch case construction developers adds default case, it's a common issue that there is no default case, but it would be nice to have something like 
```
default: 
  // Do nothing
```

or 
```
default: 
 Error.throw('there is no such a case')
```